### PR TITLE
Review of Release Notes

### PR DESCRIPTION
**Pull request by sabeena21**
### Change Description ###

Made minor modification to the content.

### CODE REVIEW

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/553

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/553

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/553

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/553

File: docs/application/tizen-studio/release-notes/3-1-release-notes.md [hunk 1]
```diff
@@ -8,23 +8,23 @@
...
-	-   Fixed the issues with log filtering in log tab.
+	-   The error message has been modified, which appears while adding an incorrect privilege file to a native application.
+	-   The issues with log filtering in log tab has been fixed.
 -   Certificate manager
-	-   Fixed the failure in generating a new certificate profile after 01 January 2019.
-	-   Fixed the issue in network proxy settings. Now, the network proxy setting change is applied for all network accesses.
-	-   Made the unnecessary input fields optional in UI while creating IoT certificates. 
+	-   The failure in generating a new certificate profile after 01 January 2019 has been fixed.
+	-   The issue in network proxy settings has been fixed. Now, the network proxy setting change is applied for all network accesses.
+	-   The unnecessary input fields has been made optional in UI while creating IoT certificates.
```
[the remark this answers is not in the document]
Comment (sabeena21) [reply docs/application/tizen-studio/release-notes/3-1-release-notes.md]: @mijin85cho, I agree with your comment. I will change it to "has been changed to optional"

File: docs/application/tizen-studio/release-notes/3-0-release-notes.md [hunk 2]
```diff
@@ -70,7 +71,7 @@
 	-   Basic Web applications does not install on SD cards.
 	-   Samsung Certificate cannot be used to sign application while launching application on 5.0 Emulator.
 -	CLI and SDB
-	-   The Tizen Studio does not support the [Smart Development Bridge](../common-tools/smart-development-bridge.md) (SDB) bash auto-completion on Windows&reg; (it is available on Ubuntu and macOS).
+	-   The Tizen Studio does not support the [Smart Development Bridge](../common-tools/smart-development-bridge.md)(SDB) bash auto-completion on Windows&reg; (it is available on Ubuntu and macOS).
```
[the remark this answers is not in the document]
Comment (sabeena21) [reply docs/application/tizen-studio/release-notes/3-0-release-notes.md]: @mijin85cho, Thanks you for pointing out about space removal.
I will make the change.

File: docs/application/tizen-studio/release-notes/2-4-release-notes.md [hunk 3]
```diff
@@ -2,39 +2,38 @@
...
 
-
 ## IDE and Tools
 
 
 ### New Features
 
 -   Device Manager
-	-   Enabled Device Manager Context UI to be extended by plugins
+	-   Enabled Device Manager Context UI to be extended by plugins.
```
[the remark this answers is not in the document]
Comment (sabeena21) [reply docs/application/tizen-studio/release-notes/2-4-release-notes.md]: @mijin85cho, I will change it.

File: docs/application/tizen-studio/release-notes/3-1-release-notes.md [hunk 4]
```diff
@@ -8,23 +8,23 @@
...
+	-   The issue in network proxy settings has been fixed. Now, the network proxy setting change is applied for all network accesses.
+	-   The unnecessary input fields has been made optional in UI while creating IoT certificates. 
 -   Device Manager
-	-   Fixed the issues with log filtering in log tab.
+	-   The issues with log filtering in log tab has been fixed.
 -   TizenRT
-	-   Fixed the issue in RT IDE . Now, the IoT certificates are copied to the project while building instead of flashing.
-	-   RT IDE installation prompts user to install libwebkitgt to avoid "browser path not set" error message.
-	-   Fixed the issues that occurs while debugging Tizen RT application. 
+	-   The issue in RTIDE has been fixed. Now, the IoT certificates are copied to the project while building instead of flashing.
```
[the remark this answers is not in the document]
Comment (sabeena21) [reply docs/application/tizen-studio/release-notes/3-1-release-notes.md]: @mijin85cho, Thanks for the answer.
There was confusion whether it is RT-IDE, RTIDE or RT IDE. I got it confirmed by the developer that it is RT IDE.
I will change it to RT IDE

File: docs/application/tizen-studio/release-notes/2-4-release-notes.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (sabeena21) [reply docs/application/tizen-studio/release-notes/2-4-release-notes.md]: @mijin85cho, I will change it

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/553

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/553

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/553

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/553

File: docs/application/tizen-studio/release-notes/2-4-release-notes.md [hunk 5]
```diff
@@ -2,39 +2,38 @@
...
+-   RT IDE
+	-   RT IDE GNU Compiler Collection (GCC) toolchain has been updated to version 6.3.1.
 -   SmartThings&trade;
-	-   The setupId is a mandatory field and is mapped to the device onboarding ID.
-	-   The RESTful API Modeling Language (RAML) for SmartThings&trade; is updated.
+	-   The setupId is a mandatory field and has been mapped to the device onboarding ID.
+	-   The RESTful API Modeling Language (RAML) for SmartThings&trade; has been updated.
 -   Package Manager
-	-   Removed portable object (*.po) files , which are packaged into tpk.
+	-   The portable object (*.po) files has been removed, which are packaged into tpk.
```
Comment (mijin85cho) [inline docs/application/tizen-studio/release-notes/2-4-release-notes.md]: Just a request to check the tense of this sentence.
"has bee pp" + "present". Is it ok?

Comment (sabeena21) [reply docs/application/tizen-studio/release-notes/2-4-release-notes.md]: @mijin85cho, Thanks for pointing it out. It is incorrect to use "has been pp" + "present"
 I will be changing the sentence as:
The portable object (*.po) files, which are packaged into tpk have been removed.

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/553

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/553

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/553

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/553

File: docs/application/tizen-studio/release-notes/2-3-release-notes.md [hunk 6]
```diff
@@ -25,18 +25,19 @@
...
- > **Note**  
- > From futural Tizen Studio version onwards, support for 32 bit host for Windows and Ubuntu will be removed.
+ > **Note** 
+ > 
+ > From futural Tizen Studio version onwards, support for 32 bits host for Windows and Ubuntu will be removed.
 
 ### Fixed Bugs
 -	Emulators
-	-   Fixed a bug in Emulator Manager that shows the deleted skin in the list of device definition.
+	-   A bug in the Emulator Manager that shows the deleted skin in the list of device definition has been fixed.
```
[the remark this answers is not in the document]
Comment (sabeena21) [reply docs/application/tizen-studio/release-notes/2-3-release-notes.md]: @mijin85cho, I must have missed out.
I will add the before Device Manager.

[the remark this answers is not in the document]
Comment (mijin85cho) [reply docs/application/tizen-studio/release-notes/2-3-release-notes.md]: @sabeena21 I'm asking in the opposite point of view. Should the "the" be added to "Emulator Manager"?

**Comment by sabeena21** [comment]
Global question : While changing to "has been pp" form, I can see "have been pp" and "has been pp" are mixed.
For example, "The validation for .pem and .der files have been added when they are imported as a new IoT profile in the Certificate Manager." in 2-5-release-notes.md. In this sentence, I think "validation " is the subject. If so, isn't "has" correct? I'm not sure if the subjects and verbs match in some updates. How about checking that point again?
Answer: Thanks for your explanation. I will fix it.

@mijin85cho, As seen in the developer.tizen.org website. "The" is used before Emulator Manager.
To maintain consistency, I have placed the before Emulator Manager.
As per grammar rule, Emulator Manager is a "proper noun" "the" must not be used.
Please can you let me know your opinion on this?

**Comment by mijin85cho** [comment]
>@mijin85cho, As seen in the developer.tizen.org website. "The" is used before Emulator Manager.
To maintain consistency, I have placed the before Emulator Manager.
As per grammar rule, Emulator Manager is a "proper noun" "the" must not be used.
Please can you let me know your opinion on this?

If so, I think we'd better remove "the" in front of Emulator Manager to follow the grammar rule.
As this #553 PR is to review the Release Notes,
in this PR, please remove "the" from "the Emulator Manager" in the files under release-notes/ only.

For "the Emulator Manager" in the files outside of release-notes/ folder, please create another PR.

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/553

File: docs/application/tizen-studio/release-notes/2-3-release-notes.md [hunk 7]
```diff
@@ -62,28 +63,28 @@
 	-   Playing sound is not supported on Windows&reg; or macOS.
 	-   The Component Designer crashes if an alias is selected as the source group of an added item.
 -	Emulator
-	-   To use the Tizen emulator, install an Intel VTx supported by the CPU, and the latest version of the graphic card driver provided by the vendor. To verify the prerequisites for the Tizen emulator, see [Prerequisites for the Tizen Studio](https://developer.tizen.org/development/tizen-studio/download/installing-tizen-studio/prerequisites).
+	-   To use the Tizen emulator, install an Intel VTx supported by the CPU, and the latest version of the graphic card driver provided by the vendor. To verify the prerequisites for the Tizen emulator, see [Prerequisites for Tizen Studio](https://developer.tizen.org/development/tizen-studio/download/installing-tizen-studio/prerequisites).
```
Comment (mijin85cho) [inline docs/application/tizen-studio/release-notes/2-3-release-notes.md]: Please change to the md file version link.

File: docs/application/tizen-studio/release-notes/3-1-release-notes.md [hunk 8]
```diff
@@ -8,36 +8,36 @@
...
-	-   Fixed the failure in generating a new certificate profile after 01 January 2019.
-	-   Fixed the issue in network proxy settings. Now, the network proxy setting change is applied for all network accesses.
-	-   Made the unnecessary input fields optional in UI while creating IoT certificates. 
+	-   The failure in generating a new certificate profile after 01 January 2019 has been fixed.
+	-   The issue in network proxy settings has been fixed. Now, the network proxy setting change is applied for all network accesses.
+	-   The unnecessary input fields have been changed to optional in UI while creating IoT certificates. 
 -   Device Manager
-	-   Fixed the issues with log filtering in log tab.
+	-   The issues with log filtering in log tab have been fixed.
 -   TizenRT
```
Comment (mijin85cho) [inline docs/application/tizen-studio/release-notes/3-1-release-notes.md]: Tizen RT? TizenRT?

Comment (sabeena21) [reply docs/application/tizen-studio/release-notes/3-1-release-notes.md]: @mijin85cho, it is TizenRT

File: docs/application/tizen-studio/release-notes/2-3-release-notes.md (hunk 7 above)
Comment (sabeena21) [reply docs/application/tizen-studio/release-notes/2-3-release-notes.md]: @mijin85cho, okay sure I will change it.

File: docs/application/tizen-studio/release-notes/3-1-release-notes.md (hunk 8 above)
Comment (mijin85cho) [reply docs/application/tizen-studio/release-notes/3-1-release-notes.md]: @sabeena21 If so, would you please fix the "Tizen RT" in line 23 from 3-0-release-notes.md?

Comment (sabeena21) [reply docs/application/tizen-studio/release-notes/3-1-release-notes.md]: @mijin85cho, thank you for pointing it out.
I have changed "Tizen RT" to "TizenRT"

**Pull request merged**